src/models/launch_heavy_3/repairs.md

```diff
@@ -0,0 +1,169 @@
+
+2. Turn the keyboard over and remove the eight bottom panel screws.
+    - The screws are set into adapters that may or may not come out with the screws.
+
+![Bottom panel screws](./img/bottom-panel-screws.webp)
+
+3. Remove the bottom panel.
+    - If the magnetic lift bars are installed, they can be used as handles.
+
+![Bottom panel removed](./img/internal-bottom.webp)
+
+4. Disconnect and remove the flat ribbon cable between the main PCB (`launch 3.2`) and the numpad PCB (`launch_heavy 1.2`).
+    - Slide the black latches away from the white connectors to free the cable.
+
+![Ribbon cable](./img/ribbon-cable.webp)
+
+5. Push each PCB up out of the chassis through one of the switch holes, then lift each PCB out of the chassis, one at a time.
+    - For the main board, to avoid the USB-C port catching on the chassis, push through one of the bottom row switch holes and lift the bottom edge of the PCB first.
+
+![Loose PCBs](./img/pcbs-removed.webp)
+
+### Steps to install the PCBs:
+
+1. Install all of the switches into the chassis.
+    - The switches can also be installed after installing the PCBs, but installing the switches first is recommended because it reduces the risk of individual bent pins damaging the PCBs or switches during installation.
+    - Make sure the switches are in the correct orientation.
+        - For the two vertical keys on the numpad (highlighted red below), the LED window should be on the right.
+        - For all other switches, the LED window should be at the top.
+    - The switches are not specific to their key/hole.
+
+![Switches installed in chassis](./img/chassis-with-switches.webp)
+
+2. Turn the chassis over and check that all of the switch pins are straight.
+    - Look down the rows of pins from the sides of the chassis to identify any bent pins.
+    - If any of the pins are bent when installing the PCBs, the switch or corresponding PCB may be damaged.
+    - In the image below, the pin marked with a red arrow is an example of a pin that is bent and needs to be straightened.
+
+![Switch pins in chassis (with a bent pin)](./img/chassis-switch-pin-check.webp)
+
+3. Carefully line up the main PCB with the switch pins and place it in the chassis, starting with the USB-C port on the top edge.
+    - Evenly press the PCB down until all of the switch pins are fully seated in their sockets.
+    - It may help to pick up the keyboard and pinch to press both the PCB and the switches at the same time.
+    - If you feel excessive resistance, stop, remove the PCB, and ensure that all of the switch pins are lined up.
+    - The switch pins can be seen from the back of the sockets when they are fully seated, highlighted below.
+    
+![Switch pins visible through sockets](./img/pcb-switch-pins.webp)
+
+4. Repeat the previous step for the numpad PCB.
+5. Insert the ribbon cable into the white connector on both PCBs, then slide the black latches shut.
```

Comment:
I would add language here to stress that the user should ensure that the cable is clear of the screw-post.

Reply:
I generally like to assume readers aren't stupid and don't need to have physically obvious things pointed out, but I agree we can specify the orientation of the ribbon since it can be installed either way, and I myself had to reference the pre-removal photo to remember the correct orientation when I was putting it back together last night.

Reply:
That's fair. I think I just look at it and... recognize that we have failed to develop the most intuitive design with that cable placement.

Reply:
I don't think the cable placement's too bad, we mechanically wanted a screw hole there and the ribbon gets out of the way for it.

Added a note.